compliance_checker/acdd.py

```diff
@@ -256,7 +256,7 @@ def check_lat_extents(self, ds):
 
         # identify lat var(s) as per CF 4.1
         lat_vars = {}  # var -> number of criteria passed
-        for name, var in ds.variables.items():
+        for _name, var in ds.variables.items():
```

Comment:
Surprised it didn't change to just `.values()` here instead.

Reply:
It probably doesn't have the notion of the tuple output just checks for unused vars. But we can change that to values.